lib/cyclopedio/syntax/name_decorator.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'stanford/all'
+require 'wiktionary/noun'
+
+module Cyclopedio
+  module Syntax
+
+    class NameDecorator < SimpleDelegator
+      def initialize(category, options={})
+        @category = category
```

Comment:
We don't have to store it as a separate field, since it is availabe via __getobj__ call. Moreover, as far as I remember we also wanted to use this method with articles...